magda-web-client/src/Components/Common/SecClassification.tsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import PropTypes from "prop-types";
+import "./SecClassification.scss";
+
+function SecClassification(props) {
+    const classification = props.secClass;
+    if (!classification) {
+        return null;
+    }
+    return (
+        <div className="classication-box">
+            <div className="dataset-heading">
+                Security Classification: {classification}
+            </div>
+        </div>
+    );
+}
+
+SecClassification.propTypes = {
```

Comment:
Can we just use a typescript typing on props instead of proptypes? There's still a few `propTypes` definitions hanging around from the pre-typescript days, but having props enforced at compile time is a bit nicer.

Reply:
Yep. I initially copied from `TagsBox` and never changed it.